apps/client/.eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+  "extends": ["plugin:@nx/react", "../../.eslintrc.json"],
```

Comment:
generalize eslint rules into one file for the project, as this is a monorepo we can have one set of rules for the whole project